Ada Examples/Examples Based On OpenGL Superbible/Simplest/src/buffers.adb

```diff
@@ -0,0 +1,51 @@
+with Ada.Exceptions; use Ada.Exceptions;
+with Ada.Text_IO; use  Ada.Text_IO;
+
+with  Interfaces.C.Pointers;
+
+with GL.Attributes;
+with GL.Objects.Buffers;
+with GL.Objects.Vertex_Arrays;
+with GL.Types; use  GL.Types;
+
+with Shaders_Program;
+
+package body Utilities is
+  --  generic package parameters of Interfaces.C.Pointers:
+  --  type Index is (<>);
+  --  type tElement is private;
+  --  type Element_Array is array (Index range <>) of aliased Element;
```

Comment:
Commented code in examples tends to confuse readers. Please remove it (also in the rest of the file).